git/internal/e2e/go.mod

```diff
@@ -55,7 +56,7 @@ require (
 	github.com/fsnotify/fsnotify v1.5.4 // indirect
 	github.com/go-git/gcfg v1.5.0 // indirect
 	github.com/go-git/go-billy/v5 v5.3.2-0.20210603175951-e0768be422ff // indirect
-	github.com/go-logr/logr v1.2.3 // indirect
+	github.com/go-git/go-git/v5 v5.4.2 // indirect
```

Comment:
Please add a redirect to use only one `go-git` version (fluxcd's) is being used.